bundles/org.openhab.binding.mqtt.generic/src/main/java/org/openhab/binding/mqtt/generic/values/PercentageValue.java

```diff
@@ -95,12 +95,13 @@ public void update(Command command) throws IllegalArgumentException {
         } else //
                // Increase or decrease by "step"
         if (command instanceof IncreaseDecreaseType) {
+            final BigDecimal stepPercent = step.multiply(HUNDRED).divide(span, MathContext.DECIMAL128);
```

Comment:
Do we have to calculate this every time?
Can't this be a member variable?

Reply:
Done